### PR TITLE
Integrate cloud-stats plugin

### PR DESCRIPTION
**Pull request by olivergondza**
Add support for cloud health monitoring as designed in https://groups.google.com/forum/#!topic/jenkinsci-dev/UifBMWDkD_o.

@jenkinsci/code-reviewers, this is a example of cloud-stats plugin integration that will be needed for every other provisioning plugin (if its activity should be tracked). Please review thoroughly before cloud-stats:0.1 is released / this is merged. Thanks.

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
Sorry, missed the PR.
There are some compatibility changes, but I assume it's fine.

File: src/main/java/jenkins/plugins/openstack/compute/JCloudsComputer.java [hunk 1]
```diff
@@ -44,6 +47,11 @@ public int getRetentionTime() {
         return getNode().getSlaveOptions().getRetentionTime();
     }
 
+    @Override
+    public @Nonnull ProvisioningActivity.Id getId() {
+        return getNode().getId();
```
Comment (oleg-nenashev) [inline src/main/java/jenkins/plugins/openstack/compute/JCloudsComputer.java]: risk of NPE if the node gets disconnected

File: pom.xml [hunk 2]
```diff
@@ -71,6 +71,11 @@
                 <artifactId>config-file-provider</artifactId>
                 <version>2.7.5</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>cloud-stats</artifactId>
+                <version>0.1-SNAPSHOT</version>
```
Comment (oleg-nenashev) [inline pom.xml]: Is there a release ETA?

File: src/main/java/jenkins/plugins/openstack/compute/JCloudsComputer.java (hunk 1 above)
Comment (olivergondza) [inline src/main/java/jenkins/plugins/openstack/compute/JCloudsComputer.java]: Yes, I should propagate it all the way to the computer.

File: pom.xml (hunk 2 above)
Comment (olivergondza) [inline pom.xml]: I already released 0.1, yesterday IIRC.

**Pull request merged**